source/extensions/extensions_build_config.bzl

```diff
@@ -61,7 +61,7 @@ EXTENSIONS = {
     #
 
     "envoy.filters.network.client_ssl_auth":            "//source/extensions/filters/network/client_ssl_auth:config",
-    "envoy.filters.network.dubbo_proxy":                "//source/extensions/filters/network/dubbo_proxy:config",
+    "envoy.filters.network.dubbo_proxy":                "//source/extensions/filters/network/dubbo_proxy:hessian_utils_lib",
```

Comment:
Is this needed? Shouldn't the unit test bring in the library and test it? Can it just be completely removed for now?

Reply:
If you don't add a line here, envoy_extension_cc_test will be no-op so the test won't be picked up by CI.

Reply:
I see, OK. make sense.